src/plugins/controls/public/control_group/editor/create_control.tsx

```diff
@@ -83,6 +85,7 @@ export const CreateControlButton = ({
               width={defaultControlWidth ?? DEFAULT_CONTROL_WIDTH}
               updateTitle={(newTitle) => (inputToReturn.title = newTitle)}
               updateWidth={updateDefaultWidth}
+              updateGrow={updateDefaultGrow}
```

Comment:
I don't think we necessarily want to remember this setting between controls, right? From my understanding of this, we should only remember the **default** control size, which is managed exclusively through the control group settings flyout. Therefore, when creating a new control, the size options **should** default to this - but changing them when creating a new control and/or editing an existing control should **not** impact the default size.

Here is a video to show what I mean when I say that creating a new control impacts the default (which, in my opinion, is a bit confusing):

https://user-images.githubusercontent.com/8698078/167458051-eb53fdc0-be16-4175-ac7c-66966540e6bb.mp4

I think this is actually the same behaviour we had for control width **before** this change, which I'm honestly surprised wasn't caught before! Perhaps @andreadelrio could provide some input here, just in case this is actually the desired behaviour :)

Reply:
**Edit:** I took care of this as part of [my PR](https://github.com/elastic/kibana/pull/131771) for removing the "Set all sizes to default" checkbox (specifically in 0696a712fe7919dcd6632c760b7242763143f539) - here's a video of the new behaviour: 

https://user-images.githubusercontent.com/8698078/167485638-127fe533-04f5-43c4-8233-ca2c22e6b35b.mp4

Personally, this new way of managing control sizes is much clearer to me - basically, adding a new control will **not** impact the default size in any way, only the "Control group settings" flyout will do that. Combined with my PR which introduces a modal that confirms whether or not the new default should be applied to existing controls when modified in the control group settings, I think it's a lot more straight forward! Obviously, this would need some discussion/agreement from everybody, though :)

Reply:
Just confirming here that the behaviour of "setting the size of the control during control creation sets the default size" was something that happened because I implemented the size check buttons and wanted to store the last used size **before** I implemented the default size picker in the settings menu, then forgot to remove the original behaviour. So it was indeed the default behaviour before this PR. 

I agree that we should separate the two, especially because the default size is saved into the dashboard saved object. So only changing the default size via the settings should actually change it.

Reply:
<img width="808" alt="Screen Shot 2022-05-16 at 10 42 17 AM" src="https://user-images.githubusercontent.com/1697105/168652216-8ad20e43-2d2e-47e0-b40b-d3e3b7be6af4.png">

As we discussed in the controls sync, I've now removed all width/expand settings from the group controls since they were causing a bit of confusion. 

I'm preserving the behavior that Hannah and Devon were describing where we store the selected width and expand settings on their last control and using those settings as the default when creating new controls. These settings will however revert back to the default `medium` width and expand `true` settings if you refresh or navigate away from the dashboard you were adding controls to.